tests/data/test_dataloader.py

```diff
@@ -91,10 +91,10 @@ def test_general_neg_sample_dataloader_in_pair_wise(self):
         for batch_data in train_data:
             batch_item_list = train_item_list[pr : pr + train_batch_size]
             assert (batch_data["item_id"].numpy() == batch_item_list).all()
-            assert (batch_data["item_id"] == batch_data["price"]).all()
+            assert (batch_data["item_id"] == batch_data["price"][..., 0]).all()
```

Comment:
If we do not use float-like normalization or discretization, can the use of floating-point data be consistent with the past? E.g., no need for `[..., 0]`?